clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/FleetController.java

```diff
@@ -579,8 +579,7 @@ public StatusPageResponse fetchStatusPage(StatusPageServer.HttpRequest httpReque
             responseCode = StatusPageResponse.ResponseCode.INTERNAL_SERVER_ERROR;
             message = "Internal Server Error";
             hiddenMessage = ExceptionUtils.getStackTraceAsString(e);
-            log.log(Level.FINE, "Unknown exception thrown for request " + httpRequest.getRequest() +
-                    ": " + hiddenMessage);
+            log.log(Level.FINE, "Unknown exception thrown for request " + httpRequest.getRequest() + ": " + hiddenMessage);
```

Comment:
Missing lazy logging and generating of stack trace

Reply:
Yes, I gave up on a very few of these :)

Reply:
Could have used the if thing. sure. Ok ok .